src/humancellatlas/data/metadata/api.py

```diff
@@ -596,13 +600,13 @@ class ManifestEntry:
     uuid: UUID4
     version: str
 
-    @classmethod
-    def from_json(cls, json: JSON):
-        kwargs = dict(json)
+    def __init__(self, json: JSON):
+        kwargs = dict(json, json=dict(json))
```

Comment:
Both copies can be avoided.

Reply:
The `kwargs` name does not makes sense after removing the constructor invocation.